mypy/test/testcheck.py

```diff
@@ -104,6 +104,8 @@
     typecheck_files.append('check-python38.test')
 if sys.version_info >= (3, 9):
     typecheck_files.append('check-python39.test')
+if sys.version_info >= (3, 10):
```

Comment:
You don't have to create a separate file for just a single test.
You can use `# flags: --python-version 3.10`